pom.xml

```diff
@@ -54,11 +54,11 @@ under the License.
 		<maven.compiler.target>${java.version}</maven.compiler.target>
 		<slf4j.version>1.7.36</slf4j.version>
 		<log4j.version>1.2.17</log4j.version>
-		<jmh.version>1.19</jmh.version>
+		<jmh.version>1.37</jmh.version>
 		<junit.version>4.13.2</junit.version>
 		<avro.version>1.11.1</avro.version>
 		<mockito.version>3.4.6</mockito.version>
-		<maven.exec.version>1.6.0</maven.exec.version>
+		<maven.exec.version>3.3.0</maven.exec.version>
```

Comment:
The old version had issues trying to log `-p` so went for the most recent release.